### ISSUE TITLE
Table_IO / QP improvements.

### ISSUE THREAD

**Issue by eacharles** [LSSTDESC/SprintWeek2024#22]
# Table_IO / QP improvements.

Discussions about 

Contacts:
Day/Time:  Tuesday
Main communication channel:  pigeons?
GitHub repo: https://github.com/LSSTDESC/tables_io/, https://github.com/LSSTDESC/qp/
Zoom room (if applicable):

## Goals and deliverable

Figuring out the scope of work for Canadian in kind contribution / any other work that people want to contribute to tables_io and qp over the next 6-12 months.

## Resources and skills needed

Anything. 

## Detailed description

Basically we want to prioritize and split up any work we want to see on the tables_io and qp packages in the next year.
